index.html

```diff
@@ -185,6 +186,17 @@
             var social  = document.getElementById('social');
             social.classList.add('displayed');
         });
+      </script>
+      <script>
+        if (window.fetch){
+            fetch("/rev/").then( r => r.text()).then( rev => {
```

Comment:
We need `fetch("/rev/{{ platform.name }}")` there so that http://localhost:8080/?platform=Android works seamlessly for instance.